libs/wingc/src/type_check.rs

```diff
@@ -3252,6 +3257,20 @@ impl<'a> TypeChecker<'a> {
 			}
 		}
 
+		if source_path == self.sdk_in_wing_dir {
+			let mut cloud_ns = self
+				.types
+				.libraries
+				.lookup_nested_str("@winglang/sdk.cloud", None)
+				.unwrap()
+				.0
+				.as_namespace_ref()
+				.unwrap();
+			for child_env in &child_envs {
+				cloud_ns.envs.push(*child_env);
+			}
+		}
```

Comment:
This code inside `type_check_dir` translates roughly to "if I'm type checking a directory, and the directory happens to be the one bundled in the wing compiler, insert me into the SDK cloud namespace inside our type system"